src/main/java/com/jagrosh/vortex/commands/settings/ResetCmd.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2018 John Grosh (john.a.grosh@gmail.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.vortex.commands.settings;
+
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jdautilities.menu.ButtonMenu;
+import com.jagrosh.vortex.Vortex;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.exceptions.PermissionException;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ *
+ * @author Michaili K (mysteriouscursor+git@protonmail.com)
+ */
+public class ResetCmd extends Command
+{
+    private final Vortex vortex;
+
+    private static final String CONFIRM_RESET_EMOJI = "\u2705"; // ✅
+    private static final String CANCEL_RESET_EMOJI = "\u274c";  // ❌
+
+    public ResetCmd(Vortex vortex)
+    {
+        this.vortex = vortex;
+        this.name = "reset";
+        this.help = "reset vortex settings/data to default";
+        this.arguments = "<automod|filters|settings|ignores|punishments|strikes|all>";
+        this.category = new Category("Settings");
+        this.guildOnly = true;
+        this.userPermissions = new Permission[]{Permission.MANAGE_SERVER};
```

Comment:
Because of how much power this command has, I think it should be limited to admins.
```suggestion
        this.userPermissions = new Permission[]{Permission.ADMINISTRATOR};
```